include/swift/AST/Decl.h

```diff
@@ -4988,6 +4988,7 @@ class AbstractFunctionDecl : public GenericContext, public ValueDecl {
 
 private:
   ParameterList *Params;
+  StringRef BodyStringRepresentation;
```

Comment:
I know this isn't ready for review yet, but this should be able to go into the anonymous union below, right?

Reply:
Yep, good call!